src/DateTimeSpan.php

```diff
@@ -44,12 +46,13 @@ public function getTo()
     }
 
     /**
-     * @param  DateTime $datetime
-     * @return boolean
+     * @param DateTime $datetime
+     *
+     * @return bool
```

Comment:
You can directly remove the PHPDoc from this method and define a return type declaration.